src/makeRobot.js

```diff
@@ -38,7 +38,64 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    goForward(number = 1) {
+      if (number > 0) {
+        this.coords.y += number;
+      }
+
+      return this;
+    },
+
+    goBack(number = 1) {
+      if (number > 0) {
+        this.coords.y -= number;
+      }
+
+      return this;
+    },
+
+    goRight(number = 1) {
+      if (number > 0) {
+        this.coords.x += number;
+      }
+
+      return this;
+    },
+
+    goLeft(number = 1) {
```

Comment:
I think step would be more descriptive name
```suggestion
    goForward(step= 1) {
```